sdk/xmake.lua

```diff
@@ -163,7 +163,7 @@ toolchain("cheriot-clang")
 		toolchain:add("cxflags", default_flags, {force = true})
 		toolchain:add("cflags", default_flags)
 		toolchain:add("cxxflags", "-std=c++20")
-		toolchain:add("cflags", "-std=c2x", {force = true})
+		toolchain:add("cflags", "-std=c23", {force = true})
```

Comment:
We might be able to remove the `{ force = true }` bit here.  xmake knows this flag exists (sigh).

Reply:
Done